tox.ini

```diff
@@ -1,16 +1,22 @@
 [tox]
 envlist =
-  py{310,311}-{asyncio,pyuv}-cov,pypy-cov
+  py{310,311}-asyncio
   checkqa
 
+[gh-actions]
+python =
+  3.10: py39
+  3.11: py310
+  pypy3: pypy3
+
 [testenv]
 extras = test
 deps =
   pytest-timeout
-  cov: pytest-cov
-  pyuv: pyuv
-setenv =
-  cov: PYTEST_ADDOPTS=--cov=. {env:PYTEST_ADDOPTS:}
+# cov: pytest-cov
+# pyuv: pyuv
+# setenv =
+#   cov: PYTEST_ADDOPTS=--cov=. {env:PYTEST_ADDOPTS:}
```

Comment:
I gave up on codecov for now. https://github.com/neovim/pynvim/pull/469/commits/68bde2069b58b369a2a87ecdade8e77d28dbe68d

```
    ERROR: Failed building wheel for pyuv
    ERROR: Could not build wheels for pyuv, which is required to install pyproject.toml-based projects
```